web-console/future-openshift/operators-overview/operators-overview.md

```diff
@@ -0,0 +1,18 @@
+# Showing Operators on Overview
+
+This documentation details how operators will be surfaced on the Status page. Rather than exposing CSV objects, the status page should now expose the CRs being managed by identified CSVs in designated groups.
+
+## Group by: Application
+![image](img/operators8-app.png)
+* In the group by Application view, operator managed CRs will be grouped together with the CSV object name as the group header (AMQ Streams).
+* In this example, the “View operator details” link will bring users to the Installed Operators > AMQ Streams > All Instances tab. The “View resources” link will bring users to the Installed Operators > AMQ Streams > Kafka details > Resources tab.
```

Comment:
Not sure if I remember correctly, this is brought up by @alimobrem (can you help chime in?). 

The `View operator details` link seems to be better to link to `Installed Operators > AMQ Streams > Overview tab` ( i.e. "https://your-cluster-url/k8s/ns/your-namespace/clusterserviceversions/amqstreams.v1.0.0/" ). So that user can see doc links, other provide APIs, and create more CRs if needed.

Reply:
@tlwu2013 Makes sense, I can update that!